libs/database/src/lib/entities/i18n-item.entity.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Model,
+  PrimaryKey,
+  Table,
+} from "sequelize-typescript";
+import { BIGINT, STRING } from "sequelize";
+
+@Table({ tableName: "i18n_items", underscored: true, paranoid: false })
+export class i18nItem extends Model<i18nItem> {
+
+  @PrimaryKey
+  @AutoIncrement
+  @Column(BIGINT.UNSIGNED)
+  override id: number;
+
+  @AllowNull
+  @Column(STRING)
+  status: string | null;
+
+  @AllowNull
+  @Column(STRING)
```

Comment:
This column has a default value of `short` at the DB level. Please reflect that in the column definition.